backend/module/chaos/experimentation/api/experimentation.go

```diff
@@ -59,15 +60,29 @@ func (s *Service) Register(r module.Registrar) error {
 }
 
 // CreateExperiments adds experiments to the experiment store.
-func (s *Service) CreateExperiments(ctx context.Context, req *experimentation.CreateExperimentsRequest) (*experimentation.CreateExperimentsResponse, error) {
+func (s *Service) CreateExperiment(ctx context.Context, req *experimentation.CreateExperimentRequest) (*experimentation.CreateExperimentResponse, error) {
 	s.createExperimentsStat.Inc(1)
 
-	err := s.experimentStore.CreateExperiments(ctx, req.Experiments)
+	// If start time is not provided, default to starting now
+	now := time.Now()
+	startTime := &now
+	if req.StartTime != nil {
+		s := req.StartTime.AsTime()
+		startTime = &s
```

Comment:
nit: combine these two lines into one line?

```suggestion
		startTime = &req.StartTime.AsTime()
```

Reply:
go doesn't permit taking the address of a function result